packages/frameworks/index.d.ts

```diff
@@ -13,4 +13,10 @@ export interface Framework {
     every?: FrameworkDetectionItem[];
     some?: FrameworkDetectionItem[];
   };
+  settings?: {
+    [key: string]: {
+      value?: string;
+      placeholder?: string;
```

Comment:
Why not explicitly type `buildCommand`, `devCommand` and `outputDirectory`?

Reply:
We'll add more and specifying everything is not that extensible, and in the future we'll probably make the whole frameworks.json file typescript to make sure we also know the type when importing.

Reply:
I think we should explicitly type them – I don't see an argument against that

Reply:
> we'll probably make the whole frameworks.json file typescript

So wouldn't it be better to be explicit? If the whole file is typescript you'll be explicit, no?